### PR TITLE
Update to Windows App SDK 1.4.230913002 & WinUIEx 2.3.2

### PR DESCRIPTION
**Pull request by Mysterious-Dev**
# PR checklist

**Quick summary of changes**

This PR update Windows App SDK to 1.4.230822000

**Which issue does this PR relate to?**

n/a

**Applies to the following platforms:**

- [x] WinUI
- [ ] WPF
- [ ] UWP

All is in the title ^^.

https://www.nuget.org/packages/Microsoft.WindowsAppSDK/1.4.230822000

https://github.com/dotMorten/WinUIEx/releases/tag/v2.3

### CODE REVIEW

**Comment by JasonWei512** [comment]
In Windows App SDK 1.4, the min/max/close caption buttons for `ExtendsContentIntoTitleBar = true` are now drawn by AppWindow rather than XAML.
https://learn.microsoft.com/en-us/windows/apps/windows-app-sdk/stable-channel#known-issues
So `TitlebarHelper` needs to be updated.

**Comment by Mysterious-Dev** [comment]
Well, I've got the beginnings of a fix for that, I'll keep looking at it over the next few days to push the changes ^^.

**Comment by Mysterious-Dev** [comment]
@bpulliam @pratikone @mikebattista

I thinks, this PR is ready for review ^^

**Comment by Mysterious-Dev** [comment]
Well I updated WASDK to 1.4.2.

BTW, any news for this PR @pratikone, @bpulliam  ?

**Comment by pratikone** [comment]
Hi, thanks for the patience. For some time, I have been on-and-off investigating and working on this crash issue. https://github.com/microsoft/TemplateStudio/issues/4707

this is the reason, I am not  been able to test this out this change because it could have ripple effects.  Also my machine state is entirely broken because of that. Given it is a iterative change, I think it can wait. We anyways need the crash to be fixed before next version of Template Studio can get released

**Comment by Mysterious-Dev** [comment]
> Hi, thanks for the patience. For some time, I have been on-and-off investigating and working on this crash issue. #4707
> 
> this is the reason, I am not been able to test this out this change because it could have ripple effects. Also my machine state is entirely broken because of that. Given it is a iterative change, I think it can wait. We anyways need the crash to be fixed before next version of Template Studio can get released

Well, now that #4707 is closed, I think we can get on with updating TemplateStudio for WASDK 1.4 ^^.

File: code/TemplateStudioForWinUICs/Templates/Proj/Default/Param_ProjectName/Helpers/TitleBarHelper.cs [hunk 1]
```diff
@@ -39,51 +39,50 @@ public static void UpdateTitleBar(ElementTheme theme)
...
-                _ => new SolidColorBrush(Colors.Transparent)
+                ElementTheme.Dark => Color.FromArgb(0x66, 0xFF, 0xFF, 0xFF),
+                ElementTheme.Light => Color.FromArgb(0x66, 0x00, 0x00, 0x00),
+                _ => Colors.Transparent
             };
 
-            Application.Current.Resources["WindowCaptionButtonStrokePointerOver"] = theme switch
+            App.MainWindow.AppWindow.TitleBar.BackgroundColor = Colors.Transparent;
+
+            Application.Current.Resources["WindowCaptionForeground"] = theme switch
```
Comment (pratikone) [inline code/TemplateStudioForWinUICs/Templates/Proj/Default/Param_ProjectName/Helpers/TitleBarHelper.cs:72]: we can remove it as it is not doing anything now

File: code/TemplateStudioForWinUICs/Templates/Proj/Default/Param_ProjectName/Helpers/TitleBarHelper.cs [hunk 2]
```diff
@@ -39,51 +39,50 @@ public static void UpdateTitleBar(ElementTheme theme)
...
 
-            Application.Current.Resources["WindowCaptionButtonStrokePressed"] = theme switch
+            Application.Current.Resources["WindowCaptionForegroundDisabled"] = theme switch
             {
-                ElementTheme.Dark => new SolidColorBrush(Colors.White),
-                ElementTheme.Light => new SolidColorBrush(Colors.Black),
+                ElementTheme.Dark => new SolidColorBrush(Color.FromArgb(0x66, 0xFF, 0xFF, 0xFF)),
+                ElementTheme.Light => new SolidColorBrush(Color.FromArgb(0x66, 0x00, 0x00, 0x00)),
                 _ => new SolidColorBrush(Colors.Transparent)
             };
```
Comment (pratikone) [inline code/TemplateStudioForWinUICs/Templates/Proj/Default/Param_ProjectName/Helpers/TitleBarHelper.cs:84]: can remove this too

**Review by pratikone — CHANGES_REQUESTED** [review]
Thanks for the patience. I have reviewed this PR. Only minor changes are needed else it looks good. Once done, it can be merged and I will trigger newer version of template studio binary

**Comment by Mysterious-Dev** [comment]
> Thanks for the patience. I have reviewed this PR. Only minor changes are needed else it looks good. Once done, it can be merged and I will trigger newer version of template studio binary

@pratikone It's done, I also updated WinUIEx to 2.3.2 ^^.

**Review by pratikone — APPROVED** [review]

**Comment by pratikone** [comment]
thanks for additional change. I have merged your PR. Will get a release binary out soon

**Comment by Mysterious-Dev** [comment]
> thanks for additional change. I have merged your PR. Will get a release binary out soon

Glad to have helped (all tests passed btw) ^^.

**Comment by Mysterious-Dev** [comment]
> thanks for additional change. I have merged your PR. Will get a release binary out soon

Glad to have helped (all tests passed btw) ^^.

**Comment by pratikone** [comment]
New version of Template Studio has been released.

**Pull request merged**